docs/syntax/images.md

```diff
@@ -1,29 +1,34 @@
 # Images
 
-Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with two differences:
+Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with the following differences:
+
 1. instead of link text, you provide an image description
 2. an image description starts with `![` not just `[`
+3. image paths must be specified as absolute paths
 
-Images can be referenced from the top-level `_static` dir or a local image dir.
+:::{note}
+If you reference an image using a relative path, it must be relative to the `toc.yml` or `docset.yml` rather than relative to the page that uses it.
```

Comment:
It can be a relative path but it can not go relative beyond the toc. 


e.g 

```
/images/
  - other.png
/solutions/
   /images/
      - observability-apm-app-landing.png`
   /page/
      - index.md
   toc.yml
```

`../images/observability-apm-app-landing.png` from `index.md` is fine because that lives in the same subtree as the `toc.yml` the page lives in. 


`../../images/other.png` lives outside the `toc.yml` subtree so is at risk of being broken if the assembler rehomes this subtree.


`

Reply:
Thanks, i've taken another go at the explanation since i think this might confuse folks